ct-app/scripts/list_required_parameters.sh

```diff
@@ -0,0 +1,23 @@
+if [ "$#" -eq 0 ]; then
+    echo "Usage: $0 <folder1> <folder2> ..."
+    exit 1
+fi
+
+all_variables=""
+
+for folder in "$@"; do
+    for file in "$folder"/*; do
+        if [ -f "$file" ]; then        
+            variable_list=$(grep -oE '\b(self\.)?params[[:alnum:]._]*\b' "$file" | awk '{sub(/(self\.)?params/, ""); sub(/^\.+/, ""); if ($0 != "") print}')
+
+            all_variables+="\n$variable_list"
+        fi
+    done
+done
+
+unique_variables=$(echo "$all_variables" | tr ' ' '\n' | sort -u)
+
+IFS=$'\n' # Set Internal Field Separator to newline to iterate over lines
+for variable in $unique_variables; do
+    echo $(echo $variable | tr a-z A-Z | tr . _)
+done
```

Comment:
Why not make this a python script?

Reply:
At some point I would like to integrate it into the pipeline so that we would know at a release cut which parameters are required without looking at the code. This is easier with a bash script

Reply:
I'd say it the same. All runners 100% have python, so it's just a matter of working without the dependencies. But as you wish.